client/src/components/PropertiesSummary.tsx

```diff
@@ -12,15 +12,50 @@ import { StringifyListWithConjunction } from "./StringifyList";
 import { SocialShareAddressPage } from "./SocialShare";
 import { withMachineInStateProps } from "state-machine";
 import { NetworkErrorMessage } from "./NetworkErrorMessage";
+import { AddressRecord } from "./APIDataTypes";
+import { I18n } from "@lingui/react";
+import { defaultLocale, SupportedLocale } from "i18n-base";
 
 type Props = withMachineInStateProps<"portfolioFound"> & {
   isVisible: boolean;
 };
 
-const generateLinkToDataRequestForm = (fullAddress: string) =>
-  `https://docs.google.com/forms/d/e/1FAIpQLSfHdokAh4O-vB6jO8Ym0Wv_lL7cVUxsWvxw5rjZ9Ogcht7HxA/viewform?usp=pp_url&entry.1164013846=${encodeURIComponent(
-    fullAddress
-  )}`;
+const DATA_REQUEST_FORM_URLS = {
+  en: "https://airtable.com/shruDUDHViainzUBB",
+  es: "https://airtable.com/shrQudDbhUVz5J83u",
+};
+
+const getDataRequestUrlFromLocale = (locale: SupportedLocale) =>
+  DATA_REQUEST_FORM_URLS[locale] || DATA_REQUEST_FORM_URLS[defaultLocale];
+
+const DataRequestButton: React.FC<{
+  address: AddressRecord;
+}> = ({ address }) => {
+  const fullAddress = encodeURIComponent(
+    `${address.housenumber} ${address.streetname}, ${address.boro}`
+  );
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <a
+          onClick={() => {
+            window.gtag("event", "data-request");
+          }}
+          href={`${getDataRequestUrlFromLocale(
+            i18n.language as SupportedLocale
+          )}?prefill_Custom+Data+Request=${i18n._(t`BUILDING:`)}+${fullAddress}`}
```

Comment:
this feels like a bit of a mouthful code-wise, I wonder if it'd be easier to read if factored out into a `generateLinkToDataRequestForm` function, like the one that got removed in this PR?

Also when I look closely at this I'm realizing that the value of the localized `BUILDING:` text should be wrapped in `encodeURIComponent`, I think?

Reply:
yes! good point